.github/workflows/codeclimate_standalone.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow that is manually triggered
+
+name: CodeClimate Standalone workflow
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: [push, pull_request, workflow_call]
```

Comment:
can you add tag over here as well please ;) ?

Reply:
I think I did (edited the file), though i'm not sure